spec/lib/devices/custom.spec.js

```diff
@@ -31,6 +31,19 @@ describe("Custom Device Functions", function() {
         var color = { red: 255, blue: 0, green: 0 };
         expect(rgb).to.be.calledWith(color);
       });
+
+      it("converts 'FF00FF' to an RGB object", function() {
+        device.color("FF0000");
+        var color = { red: 255, blue: 0, green: 0 };
+        expect(rgb).to.be.calledWith(color);
+      });
+
+      it("prints an error with invalid color", function() {
+        stub(console, "error");
+        device.color("mycolor");
+        expect(console.error)
+          .to.be.calledWith("invalid color provided", "mycolor");
+      });
```

Comment:
This test stubs `console.error` but never restores it, so it's unusable from this point on - maybe throw it up into a `beforeEach/afterEach` combination?